pkg/api/scylla/v1/types_cluster.go

```diff
@@ -64,6 +64,7 @@ type ScyllaClusterSpec struct {
 	DeveloperMode bool `json:"developerMode,omitempty"`
 
 	// cpuset determines if the cluster will use cpu-pinning for max performance.
```

Comment:
```suggestion
	// cpuset determines if the cluster will use cpu-pinning.
```
if it doesn't matter for performance I'd reword it as well

Reply:
done